lib/prepare_config.js

```diff
@@ -1,12 +1,19 @@
 var registerLoader = require('./register_loader');
-var findCwd = require('./find_cwd');
+var fs = require('fs');
 
-function prepareConfig(self, env, opts) {
-  var basedir = findCwd(opts);
+function prepareConfig(self, env) {
+  var basedir = env.cwd;
   env.require.filter(toUnique).forEach(function(module) {
     self.requireLocal(module, basedir);
   });
 
+  if (env.configPath) {
+    // resolve symlink if needed
+    if (fs.lstatSync(env.configPath).isSymbolicLink()) {
+      env.configPath = fs.realpathSync(env.configPath);
```

Comment:
I was going to ask if this belonged in this file but it was always named weird.  We weren't really "preparing" any config, we were just loading explicit requires and loaders.

Reply:
Does this solve a gulp-cli issue?

Reply:
I'm not confident about this naming. Certainly this part is just loading `opts.requires` and loader for `configPath`. Additionally I want to give the application a chance to  change `env` object here before the loading. Should we provide an empty method (like `Liftoff.prototype.prepareConfig`) and execute this loading after the method?

> Does this solve a gulp-cli issue?

This modification is not for solving any issues. If this symbolic link resolving is needed, this should be applied to `configPath` by not only cli option but also config files.

Reply:
As per the comment above, do we want to remove this?

Reply:
Yes.  All tests except a case for symbolic link have passed. It's no problem to remove this.